.docker/prod.Dockerfile

```diff
@@ -110,6 +114,7 @@ RUN addgroup --gid 1001 --system gvmd && \
 RUN mkdir -p /run/gvmd && \
     mkdir -p /var/lib/gvm && \
     mkdir -p /var/log/gvm && \
+    gvm-manage-certs -a && \
```

Comment:
This would create the same certificates for **all** users of the container image. Therefore I am not going to add this change. If you really need the generation of the certificates in this container you should try adding it to the `.docker/start-gvmd.sh` file instead. Please be aware that adding certificates should stay optional.

```suggestion
```

Reply:
Hi, thank you. I have modified it following your advice.